src/raftstore/store/snap.rs

```diff
@@ -109,35 +111,46 @@ impl Display for SnapKey {
     }
 }
 
-pub struct ApplyContext {
+#[derive(Default)]
+pub struct BuildContext {
```

Comment:
It's better to preserve those prefixes since there are `Applyxxx` and `cf_kv_count/size`.